src/mpc-hc/MainFrm.cpp

```diff
@@ -14667,13 +14686,24 @@ bool CMainFrame::OpenMediaPrivate(CAutoPtr<OpenMediaData> pOMD)
         OpenSetupWindowTitle();
         checkAborted();
 
-        int audstm = SetupAudioStreams();
+        int audstm;
+
+        if (m_loadedAudioTrackIndex != -1) {

```

Comment:
Test what happens if the stored index is larger than the number of available tracks.